### DISCUSSION TITLE
Inline font configuration

### DISCUSSION THREAD

**Discussion by AmitPr** [zed-industries/zed#14419] [Feature Requests]
Is there a way to get Zed to display different fonts for different tokens? The most relevant usecase is a different font for inlay hints, which I use in VSCode:

<img width="250" alt="image" src="https://github.com/user-attachments/assets/e4ffab27-5040-4c7e-a1b2-7b9b7a1a4251">

The ability to change the font, not just the color, for any token as defined in a theme file would be amazing.

This would be especially helpful to realize the vision of font families like [GitHub's Monaspace](https://monaspace.githubnext.com/) font.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
